src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,321 @@
+user to select the contact(s) they want to share with the site. After
+selecting what to share and clicking **Done**, the promise resolves with an
+array of contacts selected by the user.
+
+When calling `select()` you must provide an array of properties you'd like
+returned as the first parameter, and optionally whether multiple contacts can be
+selected as a second parameter.
+
+```js
+const props = ['name', 'email', 'tel'];
+const opts = {multiple: true};
+
+try {
+  const contacts = await navigator.contacts.select(props, opts);
+  handleResults(contacts);
+} catch (ex) {
+  // Handle any errors here.
+}
+```
+
+The Contacts Picker API can only be called from a [secure][secure-contexts],
+top-level browsing context, and like other powerful APIs, it requires a
+user gesture.
+
+### Handling the results
+
+The Contact Picker API returns an array of contacts, and each contact
+includes an array of the requested properties. If a contact doesn't have
+data for the requested property, or the user chooses to opt-out of sharing
+a particular property, it returns an empty array.
+
+For example, if a site requests `name`, `email`, and `tel`, and a user
+selects a single contact that has data in the name field, provides two
+phone numbers, but does not have an email address, the response returned will be:
+
+```json
+[{
+  "email": [],
+  "name": ["Queen O'Hearts"],
+  "tel": ["+1-206-555-1000", "+1-206-555-1111"]
+}]
+```
+
+{% Aside 'caution' %}
+  Labels and other semantic information on contact fields are dropped.
+{% endAside %}
+
+## Security and permissions {: #security-considerations }
+
+The Chrome team designed and implemented the Contact Picker API using the core
+principles defined in
+[Controlling Access to Powerful Web Platform Features][powerful-apis],
+including user control, transparency, and ergonomics. I'll explain each.
+
+### User control {: #security-control }
+
+Access to the users' contacts is via the picker, and it can only be called with
+a user gesture, on a [secure][secure-contexts], top-level browsing context.
+This ensures that a site can't show the picker on page load, or randomly show
+the picker without any context.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="contact-picker-user-choice.jpg" class="w-screenshot"
+       alt="Screen shot, users can choose which properties to share.">
+  <figcaption>
+    Users can choose not to share some properties. In this screenshot, the
+    user has unchecked the 'Phone numbers' button. Even though the site
+    asked for phone numbers, they will not be shared with the site.
+  </figcaption>
+</figure>
+
+There's no option to bulk-select all contacts so that users are encouraged
+to select only the contacts that they need to share for that particular
+website. Users can also control which properties are shared with the site
+by toggling the property button at the top of the picker.
+
+<div class="clearfix"></div>
```

Comment:
```suggestion
<div class="w-clearfix"></div>
```

Because we have to pull in devsite's stylesheets, this class works, but it wouldn't when we move to the v2 version because we ditch their CSS. Our styles are prefixed with `w-` to avoid clashing. We created a `w-clearfix` that you can use.

Reply:
Got it, will fix.

Reply:
fixed